go.mod

```diff
@@ -238,7 +238,7 @@ require (
 )
 
 replace (
-	github.com/metal3-io/baremetal-operator => github.com/openshift/baremetal-operator v0.0.0-20220217140404-6b1ecb71984f
-	github.com/metal3-io/baremetal-operator/apis => github.com/openshift/baremetal-operator/apis v0.0.0-20220217140404-6b1ecb71984f
-	github.com/metal3-io/baremetal-operator/pkg/hardwareutils => github.com/openshift/baremetal-operator/pkg/hardwareutils v0.0.0-20220217140404-6b1ecb71984f
+	github.com/metal3-io/baremetal-operator => github.com/openshift/baremetal-operator v0.0.0-20230824090041-d75786e0f23c
```

Comment:
```
 git branch --contains d75786e0f23c --all
  remotes/openshift/release-4.11
```

Shouldn't we use a commit from 4.13?

Reply:
I assume you want `12e53b8aeee5bb4d576e88dad5611c798928249b` instead.

Reply:
@zaneb, can you please double check this commit version comment from Dmitry before adding the `cherry-pick-approved` label from our side? thanks.

Reply:
We hadn't re-vendored BMO in ages (since 4.11, in fact) and the release branches are not the place to be getting them back in sync. I re-vendored to 4.11 to minimise the number of extraneous changes that came in with the bug fix.

Reply:
ACK @zaneb, thks